build/common.sh

```diff
@@ -793,7 +828,14 @@ function kube::release::create_docker_images_for_server() {
         ln ${binary_dir}/${binary_name} ${docker_build_path}/${binary_name}
         printf " FROM ${base_image} \n ADD ${binary_name} /usr/local/bin/${binary_name}\n" > ${docker_file_path}
 
-        local docker_image_tag=gcr.io/google_containers/$binary_name:$md5_sum
+        if [[ ${arch} == "amd64" ]]; then
```

Comment:
@ixdy The issue was probably here. Some setup scripts assumed that the name of `linux/amd64` images was `${binary_name}:${md5_sum}` while I had changed it to  `${binary_name}-${arch}:${md5_sum}` when I was thinking about other arches.

This is the way to go. When we're releasing, both kinds of `amd64` images will be pushed